Makefile

```diff
@@ -6,7 +6,10 @@ venv:
 install: venv
 	. venv/bin/activate; pip install .
 
-test-install: install
+pre-test-install:
+	@`python --version || python3 --version` || (echo "Python is not installed, please install Python 2 or Python 3"; exit 1)
```

Comment:
Travis-CI for some reason is failing: https://travis-ci.org/twilio/twilio-python/builds/252559709. For some reason, `python` is defined, but the check still fails.